config/development.json

```diff
@@ -139,6 +139,7 @@
 		"onboarding/interval-dropdown": true,
 		"onboarding/playground": true,
 		"onboarding/step-container-v2": true,
+		"onboarding/step-container-v2-migration-flow": true,
```

Comment:
Thanks for separating this so we can focus on shipping well tested quality work for the main flow first.